Assets/Scripts/Utils/Saves/Save.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Xml.Schema;
+using Buildings;
+using Inventory;
+using Managers;
+using UnityEngine;
+
+namespace Utils.Saves
```

Comment:
I think this isn't really a utility because it's only used for savegames, so maybe you could put these classes in their own folder.

Reply:
those are utility classes used for saving games. If you disagree, define Utility

Reply:
You put it to utility if the class is generally reusable (for other purposes) which savegames are not.

Reply:
they are reusable for anything that involves persistent data... resolved